Stripe/STPAnalyticsClient.m

```diff
@@ -144,12 +76,16 @@ - (instancetype)init {
     if (self) {
         NSURLSessionConfiguration *config = [STPAPIClient sharedUrlSessionConfiguration];
         _urlSession = [NSURLSession sessionWithConfiguration:config];
-        _apiUsage = [NSSet set];
+        _productUsage = [NSMutableSet set];
         _additionalInfoSet = [NSSet set];
     }
     return self;
 }
 
+- (void)addClassToAPIUsageIfNecessary:(Class)klass {
+    [self.productUsage addObject:NSStringFromClass(klass)];
```

Comment:
You might want to add some sort of lock around your reads/writes of productUsage: This has a chance of blowing up if someone initializes multiple classes at once.